src/types/index.ts

```diff
@@ -9,6 +9,7 @@ export interface IUser {
   lastName: string;
   location: string;
   joined: string;
+  banned?: IUser[]
```

Comment:
It's bad practise to extends parent interface. Please, look at IFollowing interface and make new interface for banned users

Reply:
And one more thing: do you use Prettier on save? 12 line doesn't have ";" at the end.